src/app/applications/[id]/TextAreaFormField.tsx

```diff
@@ -41,7 +43,7 @@ function TextAreaFormField({
   const isDisabled = !editable || isLoading;
 
   return (
-    <div className="rounded border p-4">
+    <div className="flex flex-col rounded border p-4">
```

Comment:
You can consider moving the label and warning text into its own component to prevent duplication. Something like this maybe
```
const BaseField = (children, other props ) =>
return (
<div className="flex flex-col rounded border p-4">
      <div className="flex flex-col">
        <div>
          <h3 className="text-lg text-primary sm:text-xl">{`${title} ${
            field.optional ? "(Optional)" : ""
          }`}</h3>
        </div>
        <div className="mt-4 flex w-full">
          {children}
        </div>
      </div>
      {validationWarning && (
        <span className="text-red-600 mt-2">{validationWarning}</span>
      )}
    </div>);
}
```

so you will have something like

<BaseField other props><textarea or some other field</BaseField>

Reply:
Indeed, some refactoring is needed in the form. I will try to address it in here https://github.com/GenomicDataInfrastructure/gdi-userportal-frontend/issues/379.